.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
Ok, lets do it: not a hard requirement. We can state we got pylint enabled, follow pep8, lines shall not exceed 80 :+1: